File: atividade_nicolas/alunos/nicolas_g2/jefferson/letras_palavras.js
import { print, input } from './io_utils.js'
 
function main(){
    var par = 0;
    var impar = 0;
    var caracteres = 0;

    var frase = input("escreva uma frase: ");

    var palavras = frase.split(" ", );

    for(let count = 0; count < palavras.length; count++){
        let contador = palavras[count].length;

        if(contador % 2 === 0){
            par++;
        }else{
            impar++;
        }
        caracteres += palavras[count].length;
    }
    print("\n\n********RESULTADO*********\n");
    print(" palavras de tamanho par: " +par);
    print("\n palavras de tamanho impar: " + impar);
    print("\n quantidade de caracteres: " + caracteres+"\n");
}

    

main();

File: atividade_nicolas/alunos/nicolas_g2/giovana_lima/letras_palavras.py
''' Receba uma frase conte e mostre:
quantas palavras de tamanho par e impar, sem espaços.'''


def main():
    frase = (input('Digite uma frase:\n')).split(' ')

    tamanhos = []

    for i in range(len(frase)):
        tamanhos.append(len(frase[i]))
        i += 1

    par = 0
    impar = 0

    print(f'\nQuantidade de caracteres cada palavra: {tamanhos}')
    for k in range(len(tamanhos)):
        if tamanhos[k] % 2 == 0:
            par += 1
        else:
            impar += 1

    print(f'\nPalavras pares: {par}.')
    print(f'Palavras impares: {impar}.')


if __name__ == '__main__':
    main()
